### ISSUE TITLE
FAIL: bitcoin failed to start

### ISSUE THREAD

**Issue by crazy999666676** [raspiblitz/raspiblitz#2341]
Hello, 

I copied the "bitcoin" folder from my Raspiblitz. With "sudo cp (source) (target) -a -r -v"
Then I copied the "bitcoin" folder to a new SSD.
Unfortunately I can't get any further with the installation.
The SD card has been flashed and has the version Raspiblitz 1.7
A log file is attached.
Thank you for your support!

[Error_Log_Raspi.odt](https://github.com/rootzoll/raspiblitz/files/6598598/Error_Log_Raspi.odt)

[Error_Log.txt](https://github.com/rootzoll/raspiblitz/files/6603868/Error_Log.txt)

**Comment by romanneuromancer** [comment]
I am using [Raspibolt](https://stadicus.github.io/RaspiBolt/) so I am not 100% sure if my idea works. Unfortunately I don't have an empty disk to try it out myself.

1. Stop your `systemd services`, unmount your disk (add `#`) and reboot your system:
  ```
  $ sudo systemctl disable bitcoind
  $ sudo systemctl disable lnd
  $ sudo nano /etc/fstab
  #UUID=[your_hdd_UUID] /mnt/ext ext4 noexec,defaults 0 0
  $ sudo shutdown -r now
  ```
 2. check if the disk is `unmounted` (no entry for `/mnt/hdd/`), get the NAME of the device (e.g. `sda1`) and check the filesystem:
   ```
  $ df -h
  $ lsblk -o UUID,NAME,FSTYPE,SIZE,LABEL,MODEL
  $ sudo fsck /dev/sda1 -f
   ```
If the disk is not mounted you can copy all the relevant data from your old disk to your new one.
To get back up and running, reverse the first steps:
  ```
  $ sudo systemctl enable bitcoind
  $ sudo systemctl enable lnd
  $ sudo nano /etc/fstab
  UUID=[your_hdd_UUID] /mnt/ext ext4 noexec,defaults 0 0
  $ sudo shutdown -r now
  ```
Make sure that your "bitcoin" user is the owner of `/mnt/hdd/`:
```
$ sudo chown -R bitcoin:bitcoin /mnt/hdd/
```

You can try to `unlink` your /home/bitcoin/.bitcoin and create the link again, if there is still an error
```
$ sudo unlink /home/bitcoin/.bitcoin
$ ln -s /mnt/hdd/bitcoin /home/bitcoin/.bitcoin/
```

**Comment by openoms** [comment]
This should be solved with v1.7.1. Closing for inactivity.
